libs/common/src/sql-db-connection/migrations/1721040044234-AddFacilityUtilisationDataTable.ts

```diff
@@ -0,0 +1,44 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddFacilityUtilisationDataTable1721040044234 implements MigrationInterface {
+  name = 'AddFacilityUtilisationDataTable1721040044234';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(`
+            CREATE TABLE "FacilityUtilisationData" (
+                "id" nvarchar(10) NOT NULL,
+                "utilisation" decimal(14, 2) NOT NULL CONSTRAINT "DF_4af84a919cb3a1ea143b4a93f00" DEFAULT 0,
+                "lastUpdatedAt" datetime2 NOT NULL CONSTRAINT "DF_5e17cd4417568f3be4c469c70cb" DEFAULT getdate(),
+                "lastUpdatedByPortalUserId" nvarchar(255),
+                "lastUpdatedByTfmUserId" nvarchar(255),
+                "lastUpdatedByIsSystemUser" bit NOT NULL CONSTRAINT "DF_c394215c914eb1852c617c54bbb" DEFAULT 0,
+                CONSTRAINT "PK_789e99214c8f2bf186c167d4dfa" PRIMARY KEY ("id")
+            )
+        `);
+    await queryRunner.query(`
+            INSERT INTO "FacilityUtilisationData" (id)
+            SELECT DISTINCT "facilityId" FROM "FeeRecord"
```

Comment:
This line is added to make data backwards compatible. We will be creating a separate script to check that the fee records in staging and prod do marry up to facility ids in the mongo database before running this migration, so it should be fine to run this in all environments by that point